src/components/PositionListItem/index.tsx

```diff
@@ -203,10 +203,7 @@ export default function PositionListItem({
 
   const removed = liquidity?.eq(0)
 
-  const containsURL = useMemo(
-    () => [token0?.name, token0?.symbol, token1?.name, token1?.symbol].some((testString) => hasURL(testString)),
-    [token0?.name, token0?.symbol, token1?.name, token1?.symbol]
-  )
+  const containsURL = hasURL(token0?.symbol) || hasURL(token1?.symbol)
```

Comment:
longer term is this something we plan to store a list in a DB or use another mechanism? i can understand that most URLs are probably not great but also it doesn't seem like 100%, and we are probably also not catching many phishing tokens that have non-url names

Reply:
Here's the original jira issue:
https://uniswaplabs.atlassian.net/browse/WEB-2802?focusedCommentId=16655

and the discussion thread:
https://uniswapteam.slack.com/archives/C03BSGBKWA0/p1674574970331509

(note the false positives mentions--determination at the time was that this was an acceptable step)

There's no plan to add a "this is a scam" data source, and I don't think that we want to start bailing water on that particular sinking ship.  Annabel and I discussed the tradeoffs here and we're going to scale the current filter back to just symbols for now.

I'm thinking of two additional measures we might implement (both are simple enough that I could add them to this draft, imo):
1. check for presence of one or both LP tokens in a token list we index already. (possible perf issue w/ coingecko? will see)
2. add an analytics call to notify us of pairs that are blocked proactively instead of waiting for CX inbound.